src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import './index.css';
+import { BrowserRouter } from 'react-router-dom';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import store from './redux/store';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <BrowserRouter>
+      <Provider store={store}>
+        <App />
+      </Provider>
+    </BrowserRouter>
+  </React.StrictMode>,
```

Comment:
- [ ] Great job with this submission, @IsmaelMastronardi :clap: :clap: However, I think there are a few things to put in order before signing off this. For good user experience - I believe, the requirement suggested some ease of search parameters to quickly get the much-needed info for users by using a name search but you seem to go with an approach that might not really be of benefit to the user - which is about how many content to display. Especially because the implementation is buggy too, that is why aside from the screenshot below, I have a [video](https://watch.screencastify.com/v/KWLNHhi5jjk4zOAOOq0g) embedded too. Kindly check them and see how to improve on these.

![image](https://github.com/IsmaelMastronardi/react-redux-capstone/assets/99715640/b259e163-25cb-4a57-852b-5fd561b6ab6a)